lib/services/common/alarmManagement.js

```diff
@@ -38,7 +37,7 @@ function raise(alarmName, description) {
     if (!alarmRepository[alarmName]) {
         alarmRepository[alarmName] = {
             name: alarmName,
-            description: description
+            description
```

Comment:
are your sure about this change?

Reply:
EsLint and Prettier are. This is standard syntax for ES6.